frontend/projects/upgrade/src/app/shared/services/common-dialog.service.ts

```diff
@@ -397,40 +416,82 @@ export class DialogService {
     return this.dialog.open(DeleteSegmentModalComponent, config);
   }
 
-  openExportDesignModal(title: string, warning: string): MatDialogRef<CommonSimpleConfirmationModalComponent, boolean> {
+  openExportFeatureFlagDesignModal(): MatDialogRef<CommonSimpleConfirmationModalComponent, boolean> {
```

Comment:
yeah I'm cool with an explicit separate method for each type of modal, i agree it's a good spot to be repetitive and keep the config options clearly defined here vs passing in the data.